core/internal/cltest/factories.go

```diff
@@ -306,31 +306,63 @@ func MustInsertInProgressEthTxWithAttempt(t *testing.T, txStore txmgr.EvmTxStore
 	return etx
 }
 
-func MustInsertUnstartedEthTx(t *testing.T, txStore txmgr.EvmTxStore, fromAddress common.Address, opts ...interface{}) txmgr.EvmTx {
-	var subject uuid.NullUUID
+func MustCreateUnstartedGeneratedTx(t testing.TB, txStore txmgr.TestEvmTxStore, fromAddress common.Address, chainID *big.Int, opts ...func(*txmgr.EvmTxRequest)) (tx txmgr.EvmTx) {
```

Comment:
Does the txStore here need to be "txmgr.TestEvmTxStore"?
Could it be changed to txmgr.EvmTxStore? To just indicate that this uses non-test interface only.

Reply:
Yup makes sense, thanks!